README.md

```diff
@@ -23,6 +23,10 @@ To get a local copy up and running follow these simple example steps.
 
 - Clone this repository to your local computer. Choose the location you prefer.
 
+run "git clone https://github.com/sumairq/school-library-ruby.git"
+run "cd /school-library-ruby"
+run "bundle install" to install dependencies
+
 ### Usage
 
 - Run the command `ruby <filename>`.
```

Comment:
- [ ] [OPTIONAL] Great job stating this for the user, however, we can make this better by just stating the file name :
`ruby app.rb`